Formula/ispc.rb

```diff
@@ -18,7 +19,7 @@ class Ispc < Formula
   depends_on "cmake" => :build
   depends_on "flex" => :build
   depends_on "python@3.10" => :build
-  depends_on "llvm@12"
+  depends_on "llvm" // Must be LLVM 13
```

Comment:
Need to use Ruby comment syntax.
```suggestion
  depends_on "llvm" # Must be LLVM 13
```

Reply:
Whoops!  That's definitely a silly mistake; thank you!